quantum/process_keycode/process_autocorrect.c

```diff
@@ -19,8 +19,24 @@
 #    include "autocorrect_data_default.h"
 #endif
 
-static uint8_t typo_buffer[AUTOCORRECT_MAX_LENGTH] = {KC_SPC};
-static uint8_t typo_buffer_size                    = 1;
+#ifdef AUTOCORRECT_MULTI_BANK
```

Comment:
Not sure this is the best way to handle it, but: 
```suggestion
#if defined(AUTOCORRECT_MULTI_BANK) && __has_include("autocorrect_data_alt.h")
```

Reply:
That's closer to what the other guard was, so I've made the change, but I don't know enough C to be sure which is preferred